src/System.Runtime/tests/Performance/System.Runtime.Performance.Tests.csproj

```diff
@@ -27,7 +27,6 @@
     <Compile Include="Perf.Int32.cs" />
     <Compile Include="Perf.IntPtr.cs" />
     <Compile Include="Perf.StringBuilder.cs" />
-    <Compile Include="..\System\String.SplitTests.cs" />
```

Comment:
This is the only place where a unit test file is being included in the perf tests. I think it is a mistake. The perf test was written using the temporary extensions methods (I believe as a mistake). It should have been testing one of the older Split overloads, so there's no need to use the extension methods.